apps/meteor/tests/data/livechat/rooms.ts

```diff
@@ -319,3 +319,11 @@ export const startANewLivechatRoomAndTakeIt = async ({
 
 	return { room, visitor };
 };
+
+export const placeRoomOnHold = async (roomId: string): Promise<void> => {
```

Comment:
what do you think we don't do this anymore? I see several of these functions being created, one for each call, this is almost what an sdk should do.

it seems to me that we are meta programming the tests, sometimes more logic is introduced into these methods and it becomes difficult to understand what a simple post call should be doing

Reply:
I'm not sure I got this as well. Are you suggesting to create a SDK for testing or perhaps use the existing SDK somehow over here?
The main reason y I added this method because I had to use the same API call on 2 different tests. So instead of rewriting the whole logic for requests, I isolated it to a method call and called the method in these 2 tests. This makes the code more readable IMO, plus creates a helper which we can re-use in the future if we want to in other tests.
![image](https://github.com/RocketChat/Rocket.Chat/assets/34130764/4bd663ab-9fb9-422a-97a1-6a71f1a5c168)